assets/src/edit-story/components/canvas/layout.js

```diff
@@ -360,6 +360,7 @@ const PageArea = forwardRef(function PageArea(
             background={background}
             isControlled={isControlled}
             isBackgroundSelected={isBackgroundSelected}
+            data-lasso-reference
```

Comment:
Does this still need a separate lasso reference now when just the `fullbleedRef` could be used?